core/tests/protractor_utils/ProfilePage.js

```diff
@@ -70,6 +79,9 @@ var ProfilePage = function() {
   };
```

Comment:
Can you wait for `interestElem` to be visible before calling `getText()` on it in `expectUserToHaveInterests` above?